recipes/_configure_outputs.rb

```diff
@@ -10,8 +10,8 @@
 if [:search_head, :forwarder, :cluster_master].include? node['splunk']['node_type']
   output_stanzas['tcpout'] = {
     'forwardedindex.0.whitelist' => '.*',
-    'forwardedindex.1.blacklist' => '_.*',
-    'forwardedindex.2.whitelist' => '(_audit|_internal)'
```

Comment:
Added [in Splunk 6.1](http://docs.splunk.com/Documentation/Splunk/6.1/ReleaseNotes/MeetSplunk#Platform_instrumentation_framework) was the [Splunk Enterprise platform Instrumentation framework](http://docs.splunk.com/Documentation/Splunk/6.2.5/Troubleshooting/Abouttheplatforminstrumentationframework). This system by default on servers logs data to a new Splunk index called _introspection. As we're clustering indexers, we want to forward this data from search heads, and forwarders on which we enable this.

The way this is implemented as well (forward all indexes except for _thefishbucket, which isn't really an index but more of a scratch pad), additionally paves the way for the possibility of team separated  _* indexes, that way we could even start dividing up _internal data from forwarders for each team, and possibly lock down access to _internal itself.